mcs_benchmark_data/pipelines/mcscript/mcscript_benchmark_transformer.py

```diff
@@ -75,7 +75,7 @@ def _transform_benchmark_sample(
                     ),
                 )
 
-                if dataset_type != DatasetType.TEST.value:
+                if dataset_type.value != DatasetType.TEST.value:
```

Comment:
You don't need .value on either side, you can compare the enumerators directly. dataset_type != DatasetType.TEST.